noir/aztec_macros/src/lib.rs

```diff
@@ -585,6 +589,20 @@ fn generate_storage_implementation(module: &mut SortedModule) -> Result<(), Azte
     Ok(())
 }
 
+// Transform a function to work with AVM bytecode
+fn transform_vm_function(
+    func: &mut NoirFunction,
+    _storage_defined: bool,
+) -> Result<(), AztecMacroError> {
+    // We want the function to be seen as a public function
+    func.def.is_open = true;
+
+    // NOTE: the line below is a temporary hack to trigger external transpilation tools
+    // It will be removed once the transpiler is integrated into the Noir compiler
+    func.def.name.0.contents = format!("avm_{}", func.def.name.0.contents);
+    Ok(())
+}
+
```

Comment:
Maybe move this below `transform_function` since its below it in the if-else in `transform_module`?